meetings/2024-10-23.md

```diff
@@ -0,0 +1,154 @@
+* MDJ: Yes, but that doesn't contradict what I said. We just need to work with what is in this table and iterate on it.
+
+### DST limbo warning
+
+* eP: some timezones will shift at the end of the month, CG meeting stays pinned to UTC
+
+### Special Topic Meetings
+
+
+#### Summary of CRDT meeting
+
+URL: https://wickie.invisible.college/solid-crdt
+
+* eP: I proposed creation of **CRDT for RDF CG** please support - https://www.w3.org/community/groups/proposed/#crdt4rdf
+* ep: Have discussed, Michiel, myself, Tim, Rahul, Niko from Nextgraph, Michael from Braid were present. Discussed CRDTS specifically w. RDF, and then other things (e.g. text, JSON, etc.). For this community have followed up with people already experience with RDF CRDTS, need to follow up with people with experience with other CRDTS. Also need to prioritise which formats we want to work with beyond RDF. There will be more expertise from other efforts as well, as lots of communities with interest.
+* 
+
+#### Reminder of E2EE meeting on Nov 5th
+
+URL: https://www.w3.org/events/meetings/b277ff65-0aad-425e-bd1d-64758cd4547a/20241022T140000/
+* MDJ: Was supposed to be yesterday, moved to Nov. 5
+
+### Vocab PRs
+
+#### Add solid:proxy to Solid Terms
+
+URL: https://github.com/solid/vocab/pull/94
+* eP: My suggestion for this and the data wallet PR, we shouldn't add tests just because someone has idea for new terms. We should only add terms that are in drafts of the spec. We had some conversations with Sarven about where the requirements should go. He put them in a comment in vocab, I don't see this as appropriate place. We need to follow up over there to move specific requirements over to the approproate draft. If normative should go in a draft not vocab comment.
+
+#### Add solid:Wallet for a Data Wallet
+
+URL: https://github.com/solid/vocab/pull/93
+* eP: This should be used in one of the drafts, and then referenced. All reqiurements will be in relevant draft.
+* TBL: Principle of what you should put on the ontology itself. Push back on ontology should be in draft, when the ontology should be the raw definition of the term. If you use range, domain, foaf:account etc. then you know what those terms looks like. So where some terms of the wallet are well-defined and you can put in things. The idea of outting the normative information in the comment, if it helps explain that is fine - you can always kick through to the draft. If you find yourself writing comments in the turtle file, or the RDF, don't waste your expressive power - instead put them into the ontology e.g. rdfs:comment. Good ontologies have comments about themselves. Put stuff in the terms where it will be useful to someone who comes accross on a random piece of information on the net, which some system is looking at in the future, where they can click on the predicate and see what it means.
+* eP: We can follow up in PR.
```

Comment:
```suggestion
* eP: This should be used in one of the drafts, and then referenced. All requirements will be in relevant draft.
* TBL: Principle of what you should put on the ontology itself. Push back on ontology should be in draft, when the ontology should be the raw definition of the term. If you use `range`, `domain`, `foaf:account`, etc., then you know what those terms look like. So where some terms of the wallet are well-defined, you can put in things. The idea of putting the normative information in the comment — if it helps explain, that is fine — you can always kick through to the draft. If you find yourself writing comments in the turtle file, or the RDF, don't waste your expressive power — instead put them into the ontology, e.g., `rdfs:comment`. Good ontologies have comments about themselves. Put stuff in the terms where it will be useful to someone who comes across a random piece of information on the net, which some system is looking at in the future, where they can click on the predicate and see what it means.
* eP: We can follow up in PR.
```